xcode/Subconscious/Shared/Components/BacklinksView.swift

```diff
@@ -9,7 +9,8 @@ import SwiftUI
 
 struct BacklinksView: View {
     var backlinks: [EntryStub]
-    var onSelect: (EntryLink) -> Void
+    var onRequestDetail: (EntryLink) -> Void
+    var onLink: (_ context: Slashlink, SubSlashlinkLink) -> Void
```

Comment:
This pattern comes up a lot in this PR: we need to know what context a link was tapped in. It could be the viewer or a transclude or a backlink which changes the actual navigation address.

Reply:
Q: how does the slashlink communicate whether context is viewer or transclude or backlink? Should we introduce an enum to encode it?

Reply:
It doesn't need to know, it just needs the address that the link sits within. We can add a wrapping enum but we would just discard that information later anyway.

Reply:
If all it needs is the Peer, we could consider passing that.